### DISCUSSION TITLE
Request for endpoint domains - SSL Decryption Exception

### DISCUSSION THREAD

**Discussion by LaxVolt** [keepassxreboot/keepassxc#8695] [Q&A] [answered]
I'm currently in the process of implementing SSL Decryption in our firewalls and I'm trying to compile a list of password manager domains so that I can create an exclusion rule.  I'm trying to avoid our firewall decrypting password manager traffic.  Can someone please provide a list of domain endpoints for KeepassXC?

**Comment by droidmonkey** [comment, marked as answer]
We have none

**Reply by LaxVolt** [reply]
Thank you
